scripts/get_metrics

```diff
@@ -72,56 +80,107 @@ def make_copy_of_data(data_dir):
     return read_copy_data_dir
 
 
-def process_storage(storage, min_ts, max_ts):
-    stats = load_metrics_from_kv_store(storage, min_ts, max_ts)
+def process_storage(storage, args):
+    stats = load_metrics_from_kv_store(storage, args.min_ts, args.max_ts, timedelta(seconds=args.step))
+
+    with open(args.output, 'w') as f:
+        columns = [
+            "timestamp",
+            "avg_looper_run_time_spent",
+            "avg_three_pc_batch_size",
+            "avg_incoming_node_message_size",
+            "avg_outgoing_node_message_size",
+            "avg_incoming_client_message_size",
+            "avg_outgoing_client_message_size",
+            "node_stack_messages_processed",
+            "client_stack_messages_processed",
+            "three_pc_batch_count",
+            "requests_count",
+            "incoming_node_messages",
+            "incoming_node_traffic",
+            "outgoing_node_messages",
+            "outgoing_node_traffic",
+            "incoming_client_messages",
+            "incoming_client_traffic",
+            "outgoing_client_messages",
+            "outgoing_client_traffic"
+        ]
+        f.write(",".join(columns))
+        f.write("\n")
+        for ts, frame in sorted(stats.frames(), key=lambda v: v[0]):
+            row = [
+                ts.replace(microsecond=0),
+                frame.get(MetricsType.LOOPER_RUN_TIME_SPENT).avg,
+                frame.get(MetricsType.THREE_PC_BATCH_SIZE).avg,
+                frame.get(MetricsType.INCOMING_NODE_MESSAGE_SIZE).avg,
+                frame.get(MetricsType.OUTGOING_NODE_MESSAGE_SIZE).avg,
+                frame.get(MetricsType.INCOMING_CLIENT_MESSAGE_SIZE).avg,
+                frame.get(MetricsType.OUTGOING_CLIENT_MESSAGE_SIZE).avg,
+                frame.get(MetricsType.NODE_STACK_MESSAGES_PROCESSED).sum / stats.timestep.total_seconds(),
```

Comment:
why do divide to time here? Do we consider all metrics per second?

Reply:
Yes, some of them are per second, it gives more consistent results when changing time windows.